ChangeLog

```diff
@@ -1,3 +1,7 @@
+2022-12-08 Max Coulter <max.coulter@xelect.co.uk>
+
+   * src/devol.cpp: Additions to crossover weights from DEoptim
```

Comment:
Nit:  Old-school file format so it wants a eight-space-TAB, or simply eight spaces.